File: BlackJack.ViewModels/Game/ResponseStopGameView.cs
using System.Collections.Generic;

namespace BlackJackViewModels.Game
{
    public class ResponseStopGameView
    {
        public string GameId { get; set; }
        public List<PlayerGameView> Bots { get; set; }
        public PlayerGameView User { get; set; }
        public int Cardsleft { get; set; }

        public ResponseStopGameView()
        {
            Bots = new List<PlayerGameView>();
            User = new PlayerGameView();
        }
    }
}

File: BlackJack/src/shared/models/Game/ResponseStopGameView.ts
import { PlayerGameView } from './PlayerGameView';
import { ResponseWinnerGameView } from './ResponseWinnerGameView';

export class ResponseStopGameView {
    gameId: string;
    bots: Array<PlayerGameView>;
    user: PlayerGameView;
    cardsleft: number;
    winner: ResponseWinnerGameView;
}
